wp-includes/class-wp-web-app-manifest.php

```diff
@@ -546,6 +554,28 @@ public function register_short_name_setting() {
 				'show_in_rest'      => true,
 			)
 		);
+		/**
```

Comment:
Since this is not a DocBlock per https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#5-2-multi-line-comments

```suggestion

		/*
```

Reply:
This seems to be the difference of: knowing some standards vs. living the standards.

Thank you, I appreciate to learn!